utils/queue_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-NOTE: CODE BASE IS COPYED FROM https://github.com/eapache/queue/blob/main/v2/queue.go, modified to make it thread safe
+NOTE: CODE BASE IS COPIED FROM https://github.com/eapache/queue/blob/main/v2/queue.go, modified to make it thread safe
 
```

Comment:
Can you rewrite the commit msg? We can keep a same msg style in this project.
eg: `fix(typo): xxxx`